{"@t":"2021-02-18T11:42:26.5030342Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14824,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-VRO6LB0","Log4NetLevel":"ERROR"}
{"@t":"2021-02-18T11:47:30.6064446Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14824,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-VRO6LB0","Log4NetLevel":"ERROR"}
{"@t":"2021-02-18T11:52:34.6744995Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14824,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-VRO6LB0","Log4NetLevel":"ERROR"}
{"@t":"2021-02-18T11:57:38.7204260Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14824,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-VRO6LB0","Log4NetLevel":"ERROR"}
{"@t":"2021-02-18T12:02:42.8360108Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14824,"ProcessName":"iisexpress","ThreadId":32,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-VRO6LB0","Log4NetLevel":"ERROR"}
{"@t":"2021-02-18T12:07:46.9417395Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Ett fel uppstod när begäran skickades. ---> System.Net.WebException: Det går inte att ansluta till fjärrservern ---> System.Net.Sockets.SocketException: Det gick inte att göra en anslutning eftersom måldatorn aktivt nekade det 127.0.0.1:80\r\n   vid System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   vid System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   vid System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut på stackspårning för interna undantag ---\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   vid System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   vid Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:rad 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":14824,"ProcessName":"iisexpress","ThreadId":29,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-VRO6LB0","Log4NetLevel":"ERROR"}
